templates/openstack/what-is-openstack.html

```diff
@@ -2,206 +2,593 @@
-      {{
-        image(
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <div class="p-logo-section__item">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/15263218-hewlett_packard.svg",
+          alt="",
+          width="146",
+          height="145",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <div class="p-logo-section__item">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/94ea495b-2018-logo-Intel.svg",
+          alt="",
+          width="146",
+          height="145",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <div class="p-logo-section__item">
+        {{ image (
           url="https://assets.ubuntu.com/v1/7dd7bfaf-2018-logo-huawei.svg",
-          alt="Huawei",
+          alt="",
+          width="145",
+          height="145",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <div class="p-logo-section__item">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/446d9353-logo-nec.svg",
+          alt="",
+          width="144",
           height="144",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <div class="p-logo-section__item">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/2ec61a24-logo-deutsche-telekom.svg",
+          alt="",
           width="144",
+          height="144",
           hi_def=True,
-          loading="lazy",
-        ) | safe
-      }}
-    </li>
-    <li class="col-2 col-small-2 col-medium-2">
-      {{
-        image(
-          url="https://assets.ubuntu.com/v1/c4c720c6-suse.png",
-          alt="SUSE",
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <div class="p-logo-section__item">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/bc0dcda4-vmware-logo.svg",
+          alt="",
+          width="145",
+          height="144",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <div class="p-logo-section__item">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/d8f890fb-logo-at%26t.svg",
+          alt="",
+          width="145",
+          height="145",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-8">
+      <p>Canonical is the founding member of the foundation, official maintainer and publisher of OpenStack packages on Ubuntu, leader of the <a class="p-link--external" href="https://docs.openstack.org/charm-guide/latest/">OpenStack Charms project</a> and one the biggest contributor to OpenStack all the time.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row p-logo-section">
```

Comment:
https://vanillaframework.io/docs/patterns/logo-section - take a look how they do it here with the `u-fixed-width` class and the logo section